examples/temperature_convertor/index.html

```diff
@@ -0,0 +1,22 @@
+<p>
+	In 1724, physicist Daniel Gabriel Fahrenheit proposed a temperature scale, the Fahrenheit scale.
+  <br>The Celsius scale, is also a temperature scale but serves as an SI derived unit, and hence used worldwide.
```

Comment:
Grammar: change to 'The Celsius scale is also a temperature scale which serves as an SI unit, and hence is used worldwide'